### DISCUSSION TITLE
Agent version 32 is not sending data to Prometheus endpoint

### DISCUSSION THREAD

**Discussion by shubhamnyaik** [grafana-cold-storage/agent#3469] [Q&A]
Hi All, 

We have installed new grafana agent and noticed no data is being reported/pushed to prometheus. 
The private ip we have for prometheus is getting resolved on a public ip. 
Hosts file is updated with domain - ip mapping. 

Regards
Shubham

**Comment by tpaschalis** [comment]
I'm moving this into our Discussions page until we see whether it's a bug, let's continue there.

**Comment by tpaschalis** [comment]
Can you provide some information around your setup, and what does your config look like? (after redacting any sensitive information)

Here's what I'd do to start debugging:
* Increase the log_level to "debug" and check if there's any hints
* Look at the Agent's own metrics on the root `/metrics` endpoint to see whether there's any metrics reporting failures on scraping or remote_writing
* Look at the Agent's `/agent/api/v1/metrics/instances` and `/agent/api/v1/metrics/targets` endpoints to verify the status of my targets
